opm/grid/cpgrid/Entity.hpp

```diff
@@ -266,6 +266,14 @@ class Entity : public EntityRep<codim>
     /// Otherwise, returns itself. 
     Entity<0> getOrigin() const;
 
+    /// getLgrElem()   ONLY FOR Entities in the leaf grid view.
+    /// Returns refined entity in level > 0 (equivalent to the leaf grid view entity), if the entity was born in any LGR.
+    Entity<0> getLgrElem() const;
+
+    /// getLgrCartesianIdx()   ONLY FOR Entities in the leaf grid view.
```

Comment:
getLevelCartesianIndex?
"\brief Get cartesian index on the level grid view"

Reply:
I did it before reading this comment. Same spirit of getLevelElem(). Thanks!